src/cpu/o3/lsq.cc

```diff
@@ -1411,6 +1465,92 @@ LSQ::SplitDataRequest::isCacheBlockHit(Addr blockAddr, Addr blockMask)
 bool
 LSQ::DcachePort::recvTimingResp(PacketPtr pkt)
 {
+    // Throttling enabled
+    if (lsq->recvRespThrottling) {
+        // Reset counters/flags on new cycle
+        if (curTick() > lsq->recvRespLastActiveTick + 1) {
+            lsq->recvRespBytes = 0;
+            lsq->recvRespCachelines = 0;
+            lsq->recvRespLastCachelineAddr = 0;
+            lsq->recvRespThrottled = false;
```

Comment:
Is it necessary to make recvRespThrottled a member variable? It looks like it could be made a local one, to be calculated from the others.